### ISSUE TITLE
P3 table5-6 Why does it take more than 180 seconds

### ISSUE THREAD

**Issue by dongzi1986** [madMAx43v3r/chia-gigahorse#154]
win10 3060ti  RAM 256G

![微信截图_20230613170953](https://github.com/madMAx43v3r/chia-gigahorse/assets/136438687/9b0597c1-8240-46a1-a81c-e0edbd20d17b)

cuda_plot_k32.exe  -C 8 -n -1 -g 0  -t D:\t\ -d E:\ -d F:\ -d G:\ -f ****  -c **** -M 120

During the drawing process, I encountered P3 table5-6 freeze phenomenon, which lasted for more than 180 seconds, and the graphics card would not work there. What is the reason for this, please help

**Comment by Cathayan0** [comment]
Use plot_sink as localhost. It will solve the problem.

**Comment by dongzi1986** [comment]
> Use plot_sink as localhost. It will solve the problem.


Yes, I have tried, but I can't use this command, please help. My ssd: D:\t\ target disk E
:\ and F:\

Which one to choose between ./chia_plot_copy and ./chia_plot_sink? please write in detail

**Comment by Cathayan0** [comment]
> > Use plot_sink as localhost. It will solve the problem.
> 
> Yes, I have tried, but I can't use this command, please help. My ssd: D:\t\ target disk E :\ and F:\
> 
> Which one to choose between ./chia_plot_copy and ./chia_plot_sink? please write in detail
> 
> I am distributing locally and do not connect to remote

I will share my command to you :)

cuda_plot_k32.exe command
start C:\XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\cuda_plot_k32.exe -C 8 -M 0 -n -1 -t X:\ -d @localhost -f XXXXXX -c XXXXXXXXXX

---------------------------------------------------------------------------------------------------------------

chia_plot_sink.exe
@ECHO OFF

setlocal
start "Chia Gigahorse" cmd.exe /K ".\chia_plot_sink.exe -- E:\ F:\ H:\ I:\ J:\ K:\ "

:EXIT
endlocal

---------------------------------------------------------------------------------------------------------------
Steps

1. Replace X with your information for your command.

2. Open your chia_plot_sink.exe first to listen from your plotter, 

3. then open cuda_plot_kXX.exe

Enjoy

:)

**Comment by Cathayan0** [comment]
> 
> 
> 
> 
> > Use plot_sink as localhost. It will solve the problem.
> 
> 
> 
> 
> 
> Yes, I have tried, but I can't use this command, please help. My ssd: D:\t\ target disk E
> 
> :\ and F:\
> 
> 
> 
> Which one to choose between ./chia_plot_copy and ./chia_plot_sink? please write in detail
> 
> 
> 
> I am distributing locally and do not connect to remote

帥哥，可以了嗎？

**Comment by dongzi1986** [comment]
> > > 使用 plot_sink 作为本地主机。它会解决问题。
> > 
> > 
> > 是的，我试过了，但是我不能使用这个命令，请帮忙。我的 ssd: D:\t\ 目标磁盘 E:\ 和 F:\
> > ./chia_plot_copy 和 ./chia_plot_sink 选哪个？请写详细
> > 我在本地分发，不连接到远程
> 
> 我会把我的命令分享给你:)
> 
> cuda_plot_k32.exe 命令 启动 C:\XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\cuda_plot_k32.exe -C 8 -M 0 -n -1 -t X:\ -d@localhost-f XXXXXX -c XXXXXXXXXX
> 
> chia_plot_sink.exe @echo离开
> 
> setlocal start "Chia Gigahorse" cmd.exe /K ".\chia_plot_sink.exe -- X:\ X:\ X:\ X:\ X:\ X:\ "
> 
> :退出 本地
> 
> 脚步
> 
> 1. 将 X 替换为您的命令信息。
> 2. 首先打开你的 chia_plot_sink.exe 来听你的绘图仪，
> 3. 然后打开cuda_plot_kXX.exe
> 
> 享受
> 
> :)

Thank you, it has been solved with your method, but the speed is relatively slow and does not affect the use

**Comment by dongzi1986** [comment]
> > > Use plot_sink as localhost. It will solve the problem.
> > 
> > 
> > Yes, I have tried, but I can't use this command, please help. My ssd: D:\t\ target disk E :\ and F:\
> > Which one to choose between ./chia_plot_copy and ./chia_plot_sink? please write in detail
> > I am distributing locally and do not connect to remote
> 
> I will share my command to you :)
> 
> cuda_plot_k32.exe command start C:\XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\cuda_plot_k32.exe -C 8 -M 0 -n -1 -t X:\ -d @localhost -f XXXXXX -c XXXXXXXXXX
> 
> chia_plot_sink.exe @echo OFF
> 
> setlocal start "Chia Gigahorse" cmd.exe /K ".\chia_plot_sink.exe -- X:\ X:\ X:\ X:\ X:\ X:\ "
> 
> :EXIT endlocal
> 
> Steps
> 
> 1. Replace X with your information for your command.
> 2. Open your chia_plot_sink.exe first to listen from your plotter,
> 3. then open cuda_plot_kXX.exe
> 
> Enjoy
> 
> :)

Thank you, it has been solved with your method, but the speed is relatively slow and does not affect the use

**Comment by Cathayan0** [comment]
It is recommended to plot multiple drives at a time. Since HDDs are around 100MB/s to 300MB/s. 2 drives are not enough and the bottleneck will be too much data waiting but too slow to write. Let's say your drive's writing speed is around 150MB/s. C8 plot is around 74.7GB. One plot will take almost 500s to write on one of the drives. If you have 3 or more destination drives, the data will be equally distributed to different drives and it will make the whole process smoother.

**Comment by dongzi1986** [comment]
> It is recommended to plot multiple drives at a time. Since HDDs are around 100MB/s to 300MB/s. 2 drives are not enough and the bottleneck will be too much data waiting but too slow to write. Let's say your drive's writing speed is around 150MB/s. C8 plot is around 74.7GB. One plot will take almost 500s to write on one of the drives. If you have 3 or more destination drives, the data will be equally distributed to different drives and it will make the whole process smoother.
> 
> Can you share both of your logs with us?


It has been solved, thank you
